providers/aws/services/ec2/ec2_ebs_snapshots_encrypted/ec2_ebs_snapshots_encrypted.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+
+from lib.check.check import Check, Check_Report
+from providers.aws.services.ec2.ec2_service import ec2_client
+
+
+class ec2_ebs_snapshots_encrypted(Check):
+    def execute(self):
+        findings = []
+        for regional_client in ec2_client.regional_clients:
+            region = regional_client.region
+            if regional_client.snapshots:
+                if 'error' in regional_client.snapshots:
+                        report = Check_Report()
+                        report.status = "ERROR"
+                        report.result_extended = f"{regional_client.snapshots}"
+                        report.region = region
+                else:
+                    for snapshot in regional_client.snapshots:
+                        if snapshot["Encrypted"]:
+                            report = Check_Report()
+                            report.status = "PASS"
+                            report.result_extended = f"EBS Snapshot {snapshot['SnapshotId']} is encrypted"
+                            report.region = region
+                        else:
+                            report = Check_Report()
+                            report.status = "FAIL"
+                            report.result_extended = f"EBS Snapshot {snapshot['SnapshotId']} is unencrypted"
+                            report.region = region
+            else:
+                report = Check_Report()
+                report.status = "PASS"
+                report.result_extended = "There are no EC2 EBS snapshots"
```

Comment:
should we call this one status_extended instead of result_extended? this is also helpful for the consolidated outputs. Also we need to include the ARN here.

Reply:
these checks are not valid, only for testing purposes. We need to define the output model format.